instrumentation/runtime-metrics/javaagent/src/main/java/io/opentelemetry/instrumentation/javaagent/runtimemetrics/RuntimeMetricsInstaller.java

```diff
@@ -27,21 +28,25 @@ public void afterAgent(AutoConfiguredOpenTelemetrySdk autoConfiguredSdk) {
     ConfigProperties config = autoConfiguredSdk.getConfig();
 
     boolean defaultEnabled = config.getBoolean("otel.instrumentation.common.default-enabled", true);
-    if (!config.getBoolean("otel.instrumentation.runtime-metrics.enabled", defaultEnabled)) {
-      return;
-    }
+    if (config.getBoolean("otel.instrumentation.runtime-metrics.enabled", defaultEnabled)) {
+
+      OpenTelemetry openTelemetry = GlobalOpenTelemetry.get();
 
-    OpenTelemetry openTelemetry = GlobalOpenTelemetry.get();
+      BufferPools.registerObservers(openTelemetry);
+      Classes.registerObservers(openTelemetry);
+      Cpu.registerObservers(openTelemetry);
+      MemoryPools.registerObservers(openTelemetry);
+      Threads.registerObservers(openTelemetry);
 
-    BufferPools.registerObservers(openTelemetry);
-    Classes.registerObservers(openTelemetry);
-    Cpu.registerObservers(openTelemetry);
-    MemoryPools.registerObservers(openTelemetry);
-    Threads.registerObservers(openTelemetry);
+      if (config.getBoolean(
+          "otel.instrumentation.runtime-metrics.experimental-metrics.enabled", false)) {
+        GarbageCollector.registerObservers(openTelemetry);
+      }
+    }
 
-    if (config.getBoolean(
-        "otel.instrumentation.runtime-metrics.experimental-metrics.enabled", false)) {
-      GarbageCollector.registerObservers(openTelemetry);
+    if (config.getBoolean("otel.jmx.enabled", true)) {
```

Comment:
These metrics don't really feel like runtime metrics. Sure they're collected from JMX, which is part of the runtime, but they're not related to the function of specific libraries / frameworks rather than the JVM itself. I think it probably makes sense to put this in a new module, perhaps `:instrumentation:jmx`.

Reply:
`:instrumentation:jmx` sounds like a good place for this functionality